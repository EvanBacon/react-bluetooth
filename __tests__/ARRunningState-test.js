import React from 'react';
import renderer from 'react-test-renderer';

import ARRunningStateIOS from '../lib/ARRunningState.ios.js';
import ARRunningStateDefault from '../lib/ARRunningState.js';

describe('ARRunningState', () => {
  describe('Platform.default', () => {
    it('renders', () => {
      const tree = renderer.create(<ARRunningStateDefault />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
  describe('Platform.ios', () => {
    it('renders', () => {
      const tree = renderer.create(<ARRunningStateIOS />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
