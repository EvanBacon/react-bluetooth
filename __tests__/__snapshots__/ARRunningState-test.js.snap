// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ARRunningState Platform.default renders 1`] = `null`;

exports[`ARRunningState Platform.ios renders 1`] = `
<View
  style={
    Array [
      Object {
        "bottom": 0,
        "justifyContent": "space-between",
        "left": 0,
        "padding": 24,
        "position": "absolute",
        "right": 0,
      },
      undefined,
    ]
  }
>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    collapsable={undefined}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "aspectRatio": 1,
          "backgroundColor": "transparent",
          "justifyContent": "center",
          "width": 56,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      />
    </View>
  </View>
</View>
`;
